### PR TITLE
docs: fix broken link and add proper tech stack

### LINKED ISSUE #280: [DOCS] Tech stack + Broken link
### Some changes needed in the README 

- The `Technologies used` section looks  like it lacks several of the tech stacks we use, please update them.
- There's a weird and bad-looking gap too between those horizontal lines, do remove those
- Fixes needed for the wrong broken link in the `Useful Links` section

### Image 1 

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Dun-sin**
# Issue  📐

- **I @Dun-sin have worked for #280 **

[put x to check the boxes]: <> (This is a comment, it will not be included)

## Guidelines 🔐

**I accept the fact that i have followed the guidelines and have not copied the codes from around the internet**

- [x] **Contribution Guidelines**
- [x] **Code of Conduct**

## Issue to be closed 🛅

- **My pull request closes #280 **

## Screenshots 📷

**Here are the pictures of changes that i have made 🔽**
image
 
  
  
---

### CODE REVIEW

**Review by IAmTamal — CHANGES_REQUESTED** [review]
Awesome work 🚀
Can we also add `express`, `MongoDB`, and `Node JS` to the list of technologies used !

**Comment by Dun-sin** [comment]
> Awesome work 🚀 Can we also add `express`, `MongoDB`, and `Node JS` to the list of technologies used !

sure!!

**Comment by Dun-sin** [comment]
> Awesome work 🚀 Can we also add `express`, `MongoDB`, and `Node JS` to the list of technologies used !

sure!!

File: README.md [hunk 1]
```diff
@@ -37,13 +37,13 @@ This might be the debut repo for you, you can contribute to this repo, solve the
 ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
 ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)
 ![Bootstrap](https://img.shields.io/badge/bootstrap-%23563D7C.svg?style=for-the-badge&logo=bootstrap&logoColor=white)
-![Netlify](https://img.shields.io/badge/netlify-%23000000.svg?style=for-the-badge&logo=netlify&logoColor=#00C7B7)
+![Vercel](https://img.shields.io/badge/Vercel-000000?style=for-the-badge&logo=vercel&logoColor=white)
```
Comment (warmachine028) [inline README.md:41]: Add these 3 items as requested by **@IAmTamal**
Node JS Express Mongo DB

**Review by warmachine028 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Dun-sin) [reply README.md:44]: Yeah, I know, was adding them before you made this review

**Pull request merged**